src/app/public/modules/search/search.component.html

```diff
@@ -32,6 +32,7 @@
       >
         <input
           class="sky-form-control sky-search-input sky-rounded-corners"
+          style="border-top-left-radius: 3px; border-bottom-left-radius: 3px;"
```

Comment:
Sadly had to be applied at the element level or have !important in the style doc, because the modern theme is loaded later and has priority in its style